src/components/loop-animation.js

```diff
@@ -43,28 +44,30 @@ AFRAME.registerComponent("loop-animation", {
       return;
     }
 
-    let clip;
-
+    let clips;
     if (clipName !== "") {
-      clip = animations.find(({ name }) => name === clipName);
+      clips = clipName.split(",").map(n => animations.find(({ name }) => name === n));
     } else {
-      clip = animations[activeClipIndex];
+      clips = [animations[activeClipIndex]];
```

Comment:
We should decide if/how we want to support this for the index based property. Just avoided doing it for now since the happy path in the blender addon right now is string based, and having numeric array properties in aframe is a bit of a pain.

Reply:
I think we'll want it for use in Spoke. We can talk more about it when we want to add this feature to Spoke.